commands/help.js

```diff
@@ -1,38 +1,48 @@
-const prefix = process.env.PREFIX
+const prefix = process.env.PREFIX;
 
 module.exports = {
-    name: 'Help',
-    description: 'List all of my commands or info about a specific command.',
-    aliases: ['commands'],
-    usage: '[command name]',
-    cooldown: 5,
-    execute(message, args) {
-        const data = [];
-        const { commands } = message.client;
-
-        if (!args.length) {
-            data.push('Here\'s a list of all my commands:');
-            data.push(commands.map(command => command.name).join(', '));
-            data.push(`\nYou can send \`${prefix}help [command name]\` to get info on a specific command.`);
-
-            return message.channel.send(data, { split: true })
-        }
-
-        const name = args[0].toLowerCase();
-        const command = commands.get(name) || commands.find(c => c.aliases && c.aliases.includes(name));
-
-        if (!command) {
-            return message.reply('That\'s not a valid command.');
-        }
-
-        data.push(`**Name:** ${command.name}`);
-
-        if (command.aliases) data.push(`**Aliases:** ${command.aliases.join(', ')}`);
-        if (command.description) data.push(`**Description:** ${command.description}`);
-        if (command.usage) data.push(`**Usage:** ${prefix}${command.name} ${command.usage}`);
-
-        data.push(`**Cooldown:** ${command.cooldown || 3} second(s)`);
-
-        message.channel.send(data, { split: true });
-    },
-};
\ No newline at end of file
+  name: 'help',
+  description: 'List all of my commands or info about a specific command.',
+  aliases: ['commands'],
+  usage: '[command name]',
+  cooldown: 5,
+  execute(message, args) {
+    const data = [];
+    const { commands } = message.client;
```

Comment:
To avoid the error of use spaces after the '{' you can add a comment before the line: `// eslint-disable-next-line object-curly-spacing`